packages/documentation/copy/en/reference/Decorators.md

```diff
@@ -6,10 +6,19 @@ oneline: TypeScript Decorators overview
 translatable: true
 ---
 
+<blockquote class='bg-reading'>  
+  
+  <p>
+    This documents refer to an experimental stage 2 decorators implementation. Stage 3 decorator support is available since Typescript 5.<br>
+    See: <a href='https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#decorators'>Decorators in Typescript 5</a>
+  </p>
+</blockquote>
+
 ## Introduction
 
-<blockquote class='bg-reading'>
-  <p>Further Reading:<br /><a href='https://saul-mirone.github.io/a-complete-guide-to-typescript-decorator/'>A Complete Guide to TypeScript Decorators</a></p>
+
+<blockquote class='bg-reading'>  
+  <p>Further Reading (stage 2):<br /><a href='https://saul-mirone.github.io/a-complete-guide-to-typescript-decorator/'>A Complete Guide to TypeScript Decorators</a></p>
 </blockquote>
 
 With the introduction of Classes in TypeScript and ES6, there now exist certain scenarios that require additional features to support annotating or modifying classes and class members.
```

Comment:
I can't put a comment there, but there's a line below that mentions "stage 2 proposal", can you just remove this line?

Reply:
done